🐛 Bug: Invalid redirection URL

### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

Ends up in an infinite loop

REDIRECT_AFTER_SIGNIN

```
views.ts:178 Error: Request failed with status code 400
    at createError (createError.js:16:15)
    at settle (settle.js:17:12)
    at XMLHttpRequest.handleLoad (xhr.js:62:7)
```

### Desired Behavior

No error

### Project Details

v0.202.5

### Attachments

_No response_

--- a/packages/nc-gui/pages/index.vue
+++ b/packages/nc-gui/pages/index.vue
@@ -93,7 +93,7 @@ onMounted(() => {
     return navigateTo(route.value.query.continueAfterSignIn as string)
   } else {
     const continueAfterSignIn = localStorage.getItem('continueAfterSignIn')
-
+    localStorage.removeItem('continueAfterSignIn')
     if (continueAfterSignIn) {
       return navigateTo({
         path: continueAfterSignIn,

